rdrf/rdrf/events/events.py

```diff
@@ -2,6 +2,7 @@ class EventType:
     OTHER_CLINICIAN = "other-clinician"
     NEW_PATIENT = "new-patient"
     NEW_PATIENT_PARENT = "new-patient-parent"
+    NEW_PATIENT_ADDED = "new-patient-added"
```

Comment:
This notification will only be sent when `PATIENTS_CREATE_USERS` is enabled, meaning that the `NEW_PATIENT_ADDED` notification won't necessarily be sent when a new patient is added.

I thing renaming to `NEW_PATIENT_USER_ADDED` makes more sense than sending a notification when a patient has been added regardless of configuration. 

The latter option is something that registries haven't asked for yet and the template data would be different depending on whether  `PATIENTS_CREATE_USERS` is enabled.